orchestration/dagster_orchestration/hca_orchestration/solids/load_hca/data_files/load_data_files.py

```diff
@@ -170,13 +170,17 @@ def bulk_ingest(control_file_path: str) -> Nothing:
     job_id = run_bulk_file_ingest(control_file_path)
     wait_for_job_completion(job_id)
     check_data_ingest_job_result(job_id)
+    return job_id
 
 
-@composite_solid
-def import_data_files(scratch_dataset_name: HcaScratchDatasetName) -> Nothing:
+@composite_solid(
+    output_defs=[DynamicOutputDefinition(name="result", dagster_type=JobId)]
```

Comment:
This is returning the JobId so we can introduce the dependency between `import_data_files` and `file_metadata_fanout`.